build/yamls/antrea-windows-containerd-with-ovs.yml

```diff
@@ -22,6 +22,135 @@ data:
     & antrea-agent --config=$mountPath/etc/antrea/antrea-agent.conf --logtostderr=false --log_dir=c:/var/log/antrea --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=0
   Run-AntreaOVS-Containerd.ps1: |
     $ErrorActionPreference = "Stop"
+    $AntreaVersion = "3.0.5-antrea.0"
```

Comment:
Shld prefer to keep the driver installation steps mentioned here in yaml in a seperate .ps1 script say install-ovs-driver.ps1 and execute from yaml ( if file needs to be present prior to antrea-agent pod getting in running state) then need to download and run.

Reply:
I have all the steps related to running ovs in a seperate script 'start_ovs_windows_containerd.ps1' already present in image